testsuite/patching/pom.xml

```diff
@@ -55,6 +55,20 @@
             <artifactId>guava</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>failureaccess</artifactId>
```

Comment:
Could you also add `<scope>test</scope>`?

Reply:
Added